### ISSUE TITLE
[Bug] Invalid Int32 (ArgumentError) when navigating to channel

### ISSUE THREAD

**Issue by Jeel-Shah** [omarroth/invidious#2468]
**Describe the bug**

The logs report that there is an invalid Int32 when navigating to a channel.

**Steps to Reproduce**
1. Go to `https://yewtu.be/channel/UCJekgf6k62CQHdENWf2NgAQ`
2. See error

**Logs**
```
Title: `Invalid Int32:  (ArgumentError)`
Date: `2021-10-06T18:48:16Z`
Route: `/channel/UCJekgf6k62CQHdENWf2NgAQ`
Version: `2021.10.03-d66129b @ master`
<details>
<summary>Backtrace</summary>
<p>
  
Invalid Int32:  (ArgumentError)
  from /usr/lib/crystal/core/string.cr:432:5 in 'to_i32'
  from /usr/lib/crystal/core/pointer.cr:130:6 in 'decode_length_seconds'
  from src/invidious/helpers/extractors.cr:75:26 in 'process'
  from src/invidious/helpers/extractors.cr:532:17 in 'extract_item'
  from src/invidious/helpers/extractors.cr:556:28 in 'extract_items'
  from src/invidious/helpers/helpers.cr:250:15 in 'extract_videos'
  from src/invidious/channels/videos.cr:72:19 in 'get_60_videos'
  from src/invidious/routes/channels.cr:38:22 in 'videos'
  from lib/kemal/src/kemal/route.cr:13:9 in '->'
  from /usr/lib/crystal/core/primitives.cr:266:3 in 'call'
  from /usr/lib/crystal/core/http/server/handler.cr:28:7 in 'call'
  from /usr/lib/crystal/core/http/server/handler.cr:28:7 in 'call_next'
  from lib/kemal/src/kemal/filter_handler.cr:21:7 in 'call'
  from /usr/lib/crystal/core/http/server/handler.cr:28:7 in 'call_next'
  from /usr/lib/crystal/core/http/server/handler.cr:28:7 in 'call_next'
  from /usr/lib/crystal/core/http/server/handler.cr:28:7 in 'call_next'
  from /usr/lib/crystal/core/http/server/handler.cr:28:7 in 'call_next'
  from /usr/lib/crystal/core/http/server/handler.cr:28:7 in 'call_next'
  from src/invidious/helpers/static_file_handler.cr:167:15 in 'call'
  from /usr/lib/crystal/core/http/server/handler.cr:28:7 in 'call'
  from /usr/lib/crystal/core/http/server/handler.cr:28:7 in 'call'
  from /usr/lib/crystal/core/http/server/handler.cr:28:7 in 'call_next'
  from lib/kemal/src/kemal/init_handler.cr:12:7 in 'process'
  from /usr/lib/crystal/core/http/server.cr:500:5 in '->'
  from /usr/lib/crystal/core/primitives.cr:266:3 in 'run'
  from /src/boringssl/src/google-boringssl-251b516/ssl/internal.h:334:8 in '???'

</p>
</details>
```
**Screenshots**

**Additional context**
 - Browser (if applicable): Firefox 92.0.1 (64-bit)
 - OS (if applicable):

**Comment by xortuna** [comment]
Seems like this is on any channel with a scheduled video

**Comment by SamantazFox** [comment]
Should be fixed by #2454
